### PR TITLE
add NLS_DATE_FORMAT for Oracle

### PR DESCRIPTION
**Pull request by zalmane**

### CODE REVIEW

File: core/src/datayoga_core/blocks/relational/utils.py [hunk 1]
```diff
@@ -71,9 +71,19 @@ def get_engine(connection_name: str, context: Context, autocommit: bool = True)
...
         **extra)
 
+    if db_type == DbType.ORACLE:
+        # add NLS_DATE_FORMAT for Oracle to explicitly set the date format to ISO
+        sa.event.listen(engine, "connect", alter_session_on_oracle_connect)
+
     return engine, db_type
 
 
+def alter_session_on_oracle_connect(dbapi_connection, connection_record):
```
Comment (spicy-sauce) [inline core/src/datayoga_core/blocks/relational/utils.py]: add types

Comment (spicy-sauce) [inline core/src/datayoga_core/blocks/relational/utils.py]: add docstring (probably move line 75 here)

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: core/src/datayoga_core/blocks/relational/utils.py (hunk 1 above)
Comment (zalmane) [reply core/src/datayoga_core/blocks/relational/utils.py]: done

Comment (zalmane) [reply core/src/datayoga_core/blocks/relational/utils.py]: done

**Review by spicy-sauce — APPROVED** [review]

**Pull request merged**